gulpfile.js

```diff
@@ -1,6 +1,13 @@
 const gulp = require("gulp");
 const sass = require("gulp-sass");
+const minify = require("gulp-minify");
 
+gulp.task("compress", () => {
```

Comment:
Could but we also need a default task as well

Reply:
No